config/database.yml

```diff
@@ -1,6 +1,8 @@
 development:
   adapter: postgresql
-  database: discourse_development
+  database: <%= ENV['DISCOURSE_DB_NAME'] || 'discourse_development' %>
```

Comment:
Wouldn't this break people who modify database.yml for development? I'm assuming the current setup exists for a reason...